src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/on_driver_distraction_notification.cc

```diff
@@ -144,6 +158,32 @@ void OnDriverDistractionNotification::Run() {
   std::for_each(applications.begin(), applications.end(), processor);
 }
 
+bool OnDriverDistractionNotification::AddLockScreenDismissalWarningToMessage(
+    ns_smart_device_link::ns_smart_objects::SmartObjectSPtr notification_so,
+    uint32_t app_id) {
+  auto application = application_manager_.application(app_id);
+
+  if (!application) {
+    return false;
+  }
+
+  const auto mobile_language =
+      MessageHelper::MobileLanguageToString(application->ui_language());
```

Comment:
@IGapchuk if the only thing that we need is ui)language, I would prefere to pass language instead of app_id

Reply:
@LuxoftAKutsan Fixed in 048fe1415fce72016340c817c56367238ee052f6